src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    data-qa="big"
+    class="form"
+  >
+      <div class="form__case">
+        <input
+        type="text"
+        class="form__input"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        >
```

Comment:
```suggestion
        <input
          type="text"
          class="form__input"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```
Consider fixing indents according to the code style guide